### PR TITLE
Add info related to Blockchain's main goal

### PR DESCRIPTION
**Pull request by matextrem**
This PRs add a link to a video in order to understand the function of Blockchain technology  , which I think is quite useful for a newcomer. 
Besides, I have added a post related to features and differences between Ethereum and Bitcoin.

### CODE REVIEW

**Comment by fvictorio** [comment]
Hi @matextrem, thanks for the PR. The Ethereum vs. Bitcoin article seems useful, but I'm not sure about the video. This learning path is meant for people that want to learn to develop in Ethereum (admittedly, the name of the repo and the introduction are not very clear about this) and the video seems directed to other kind of public. At the very least, I wouldn't have it as the first item. Maybe we can add a "Talks" section and include it there?

cc @pablofullana @mariano-aguero

**Comment by pablofullana** [comment]
Indeed, looks to me that Bettina Warburg's TED Talks targets a broader audience which is kind of different from the specific one we are targeting here.
On the other hand, I like the post on Bitcoin vs. Ethereum, quite a great synopsis.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ with:
 - Digital signatures
 - Cryptographic hashing
 
+## Blockchain main goal
```
Comment (pablofullana) [inline README.md]: Blockchain-level material looks like a great addition!

File: README.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ with:
...
+Watch [this video](https://www.youtube.com/watch?v=RplnSVTzvnU) (Understanding Blockchain function). (Est. time: 15 minutes)
```
Comment (pablofullana) [inline README.md]: Please refer to other comments on this TED Talk video feedback.

File: README.md [hunk 3]
```diff
@@ -67,6 +70,9 @@ on smart contracts development. (Est. time: 15 minutes)
 ## Examples
 - [Solidity By Example](http://solidity.readthedocs.io/en/latest/solidity-by-example.html) - Contract examples from the official docs.
 
+## Ethereum vs. Bitcoin
```
Comment (pablofullana) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (matextrem) [reply README.md]: Could you give me more details ? I don't understand what you said.

Comment (pablofullana) [reply README.md]: Just making reference to my previous comment. No worries.

File: README.md [hunk 4]
```diff
@@ -12,8 +12,10 @@ with:
 - Digital signatures
 - Cryptographic hashing
 
-## Blockchain main goal
-Watch [this video](https://www.youtube.com/watch?v=RplnSVTzvnU) (Understanding Blockchain function). (Est. time: 15 minutes)
+## Blockchain
+
+1. Read [What is Blockchain?](https://medium.com/coinmonks/blockchain-technology-explained-7d68c4435aca). (Est.time: 10 minutes)
+2. Watch [this video](https://www.youtube.com/watch?v=_160oMzblY8). (Est. time: 18 minutes)
```
Comment (pablofullana) [inline README.md]: Nice. Personal opinion (non-blocker) why don't we add the video title/name to this link?

File: README.md [hunk 5]
```diff
@@ -107,3 +109,7 @@ List of tools curated by Protofire team to better apply best practices to the Pr
 - [Reddit: /r/ethereum/](https://www.reddit.com/r/ethereum/) - Reddit about next-generation platform for decentralised applications.
 - [Reddit: /r/ethdev](https://www.reddit.com/r/ethdev) - Reddit about ethereum-related dev talk: Contracts, DApps, Wallets, Clients, Infrastructure, Tooling, UIs, Patterns, and others.
 - [Ethereum Research](http://ethresear.ch/) - Civilized discussion furthering Ethereum research
+
+#### Talks
```
Comment (pablofullana) [inline README.md]: A few items here:
1. I think this doesn't belong to 'Toolbox'.
1. I tend to believe that if we keep adding categories, we'll end up having a ton of categories with just a few items in each of them. Why don't we add a 'Miscellaneous' category and that's it?

@fvictorio @mariano-aguero thoughts?

Comment (mariano-aguero) [reply README.md]: Mmm I agree to go ahead with  `Miscellaneous` category, maybe in another PR we can improve the readme an add other items like `Ethereum vs. Bitcoin` or `Under the hood` inside `Miscellaneous` (thinking out loud)

Comment (matextrem) [reply README.md]: @pablofullana  @mariano-aguero 
It sounds good to me. So, should i change 'Toolbox' category by 'Miscellaneous' or just create a miscellaneous category with the 'talks' section? 

Thank you so much for the feedback!

File: README.md (hunk 4 above)
Comment (matextrem) [reply README.md]: Perfect. I agree.

**Comment by pablofullana** [comment]
@fvictorio thoughts on this one?

**Pull request merged**